### ISSUE TITLE
Add documentation around common causes for some of the HTTP failure status codes from Community Repository

### ISSUE THREAD

**Issue by corbob** [chocolatey/docs#426]
## What New Or Updated Would You Like To See?

Add some documentation, perhaps under https://docs.chocolatey.org/en-us/community-repository/maintainers/ to list some common HTTP failure status codes (403, 409, 429 (technically already documented on rate limiting) come to mind) and what could possibly cause them.

## Why Is It Needed?

Currently there doesn't appear to be any pages that could explain potential causes for the various errors when pushing packages.

## Additional Context?

This appears to be a common question on the Discord, so would be good to get the various causes out of peoples heads into a page that we could point people to.

**Comment by corbob** [comment]
Some possible causes for codes (mostly for me to find the next time it comes up in discord):

403 could be: pushing an id that you're not the owner of.

409 could be: pushing an already approved version of existing package, or description too long, or title too long

**Comment by BenjaminMichaelis** [comment]
I agree! This caused me lots of time of going in circles trying to guess as to what the problem is! Would be very useful to have additional documentation around this.

**Comment by martin-rueegg** [comment]
After crawling through some web-search results I finally ended up on this issue (via chocolatey/choco#2800).

I'm wondering ho difficult it would be to simply return the reason in the http response message that will then be displayed as output of the command. That message could of course include a link to a documentation page for further reference.
